<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1061</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vLayoutFenetre">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hLayoutBarreOutils">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="btPrecedent">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string comment="Page précédente" extracomment="Page précédente">Page précédente</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/prec.png</normaloff>icones/prec.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRepeatDelay">
          <number>0</number>
         </property>
         <property name="autoRepeatInterval">
          <number>0</number>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSuivant">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/suiv.png</normaloff>icones/suiv.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btActualiser">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/actu.png</normaloff>icones/actu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btStopChargement">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/stop.png</normaloff>icones/stop.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btHome">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/home.png</normaloff>icones/home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="barreURL">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btAllerURL">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icones/go.png</normaloff>icones/go.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayoutTab">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrirOnglet"/>
    <addaction name="actionFermerOnglet"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionPageprecedente"/>
    <addaction name="actionPageSuivante"/>
    <addaction name="actionActualiser"/>
    <addaction name="actionArreter"/>
    <addaction name="actionAccueil"/>
    <addaction name="actionAllerURL"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAProposDezNavigo"/>
    <addaction name="actionAProposDeQt"/>
   </widget>
   <widget class="QMenu" name="menuHistorique">
    <property name="tearOffEnabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Historique</string>
    </property>
    <addaction name="actionViderHistorique"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Editer</string>
    </property>
    <addaction name="actionChercher"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEditer"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuHistorique"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrirOnglet">
   <property name="text">
    <string>Nouvel onglet</string>
   </property>
  </action>
  <action name="actionFermerOnglet">
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionPageprecedente">
   <property name="icon">
    <iconset>
     <normaloff>icones/prec.png</normaloff>icones/prec.png</iconset>
   </property>
   <property name="text">
    <string>Page précédente</string>
   </property>
  </action>
  <action name="actionPageSuivante">
   <property name="icon">
    <iconset>
     <normaloff>icones/suiv.png</normaloff>icones/suiv.png</iconset>
   </property>
   <property name="text">
    <string>Page suivante</string>
   </property>
  </action>
  <action name="actionActualiser">
   <property name="icon">
    <iconset>
     <normaloff>icones/actu.png</normaloff>icones/actu.png</iconset>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </action>
  <action name="actionArreter">
   <property name="icon">
    <iconset>
     <normaloff>icones/stop.png</normaloff>icones/stop.png</iconset>
   </property>
   <property name="text">
    <string>Arrêter</string>
   </property>
  </action>
  <action name="actionAccueil">
   <property name="icon">
    <iconset>
     <normaloff>icones/home.png</normaloff>icones/home.png</iconset>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </action>
  <action name="actionAllerURL">
   <property name="icon">
    <iconset>
     <normaloff>icones/go.png</normaloff>icones/go.png</iconset>
   </property>
   <property name="text">
    <string>Aller URL</string>
   </property>
  </action>
  <action name="actionAProposDezNavigo">
   <property name="text">
    <string>A propos de zNavigateur</string>
   </property>
  </action>
  <action name="actionAProposDeQt">
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionViderHistorique">
   <property name="text">
    <string>Vider l'historique</string>
   </property>
  </action>
  <action name="actionChercher">
   <property name="text">
    <string>Chercher</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
